superset/viz.py

```diff
@@ -145,6 +145,10 @@ def run_extra_queries(self):
         """
         pass
 
+    def handle_nulls(self, df):
+        fillna = self.get_fillna_for_columns(df.columns)
+        df = df.fillna(fillna)
```

Comment:
Maybe `df.fillna(fillna, inplace=True)`.

Reply:
It's effectively the same, are there perf implications?

Reply:
Also just moving original code to under a function